guides/source/asset_pipeline.md

```diff
@@ -605,7 +605,7 @@ config.assets.debug = false
 ```
 
 When debug mode is off, Sprockets concatenates and runs the necessary
-preprocessors on all files. With debug mode turned on the manifest above would
+preprocessors on all files. With debug mode turned off the manifest below would
```

Comment:
“Above” is correct. This sentence is referring to the manifest starting on line 558. The code sample below is not a manifest.